### PR TITLE
Updates search results for D8

### PR DESCRIPTION
**Pull request by designerneil**
## JIRA Ticket(s)
- [EWL-5905: A1 Search - Theme Search Results page](https://issues.ama-assn.org/browse/EWL-5905)

## Description:
updates checkbox and search results css to work in D8

## To Test
- [ ] visit https://americanmedicalassociation.github.io/ama-style-guide-2/?p=pages-search-results
- [ ] `gulp serve`
- [ ] does your local branch look like it? http://localhost:3000/?p=pages-search-results
- [ ] Did you test in IE 11?

## Relevant Screenshots/GIFs
screen shot 2018-10-09 at 3 13 26 pm

## Remaining Tasks
n/a

## Additional Notes
Remember this PR is related to a PR in D8
https://github.com/AmericanMedicalAssociation/ama-d8/pull/872

---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/assets/scss/02-molecules/_filter-list.scss [hunk 1]
```diff
@@ -40,6 +40,7 @@
     display:block;
 
     .ui-menu-item {
+
```
Comment (JaeBea) [inline styleguide/source/assets/scss/02-molecules/_filter-list.scss:43]:
```suggestion

```

File: styleguide/source/assets/scss/03-organisms/_main-navigation.scss [hunk 2]
```diff
@@ -72,30 +72,6 @@
     }
   }
 
-  .ama__global-search {
```
Comment (JaeBea) [inline styleguide/source/assets/scss/03-organisms/_main-navigation.scss:75]: Why is this block of css being removed?

**Review by JaeBea — CHANGES_REQUESTED** [review]
@designerneil ,
I see styles in here for the global header search. Was that originally changed on this PR's branch or is this branching off off another branch?

File: styleguide/source/assets/scss/03-organisms/_main-navigation.scss (hunk 2 above)
Comment (designerneil) [reply styleguide/source/assets/scss/03-organisms/_main-navigation.scss:75]: It was moved to _global-search.scss

File: styleguide/source/assets/scss/02-molecules/_filter-list.scss (hunk 1 above)
Comment (designerneil) [reply styleguide/source/assets/scss/02-molecules/_filter-list.scss:43]: what is the suggested changed here?

**Comment by designerneil** [comment]
@JaeBea I did merge the global header PR into this branch.
https://github.com/AmericanMedicalAssociation/ama-style-guide-2/pull/442

PR #442 needs to be merged before this one.

**Pull request merged**